packages/esm-patient-chart-app/src/ui-components/action-menu.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useCallback } from "react";
+import {
+  Edit20,
```

Comment:
Can you please use direct submodule imports? Taking the whole icons-react lib will result in a potential unnecessary increase in bundle size.

Reply:
i actually had no idea how to do that for carbon icons... can you show me an example?

Reply:
Pattern:

```js
import <AppropriateNameForTheDefaultExport> from '@carbon/icons-react/es/<name of icon>/<size of icon>';
```

Example:

```js
import ChevronDown16 from '@carbon/icons-react/es/chevron--down/16
```